test/e2e/windows/memory_limits.go

```diff
@@ -105,31 +104,66 @@ func overrideAllocatableMemoryTest(f *framework.Framework, allocatablePods int)
 		podType = "memory_limit_test_pod"
 	)
 
-	totalAllocatable := getTotalAllocatableMemory(f)
-
-	memValue := totalAllocatable.Value()
-	memPerPod := memValue / int64(allocatablePods)
-	ginkgo.By(fmt.Sprintf("Deploying %d pods with mem limit %v, then one additional pod", allocatablePods, memPerPod))
-
-	// these should all work
-	pods := newMemLimitTestPods(allocatablePods, imageutils.GetPauseImageName(), podType, strconv.FormatInt(memPerPod, 10))
```

Comment:
looking at the recent test failure the `newMemLimitTestPods` and `getTotalAllocatableMemory` are no longer used and need to be deleted.

Reply:
@jsturtevant Yes true,

Could you please help me with this error
-  test/e2e/windows/memory_limits.go:132:3: `ineffectual assignment to pod (ineffassign)`.

Reply:
from https://github.com/gordonklaus/ineffassign

>An assignment is ineffectual if the variable assigned is not thereafter used.

you are assigning pod but it's not used again. you can use `_` instead as an example.